app/src/androidTest/java/net/skyscanner/backpack/card/BpkCardTest.kt

```diff
@@ -32,6 +32,7 @@ class BpkCardTest : BpkSnapshotTest() {
   @Before
   fun setup() {
     setDimensions(150, 150)
+    setBackground(android.R.color.transparent)
```

Comment:
what's the reason for setting some of these as transparent?

Reply:
It's hard to see a white card on a white background :)

Reply:
Maybe we could use a colour that isn't in our colour palette at all to avoid clashes

Reply:
Yeah, I was also thinking about that :) But it has to be semantic to be contrast enough in the DM as well, so it just creates some extra colour management. With transparent BG it looks fine out of the box though both in the LM and the DM.